it/test/controllers/optIn/ChooseTaxYearControllerISpec.scala

```diff
@@ -17,16 +17,21 @@
 package controllers.optIn
 
 
-import controllers.optIn.ChooseYearControllerISpec.{description1Text, headingText, taxYearChoiceOne, taxYearChoiceTwo}
+//import controllers.optIn.ChooseYearControllerISpec.{description1Text, headingText, taxYearChoiceOne, taxYearChoiceTwo}
```

Comment:
Can this commented out import be removed?

Reply:
done